File: projects/DemoApp/app/src/main/java/com/astir_trotter/atcustom/demoapp/activity/SplashActivity.java
package com.astir_trotter.atcustom.demoapp.activity;

import android.support.annotation.Nullable;

import com.astir_trotter.atcustom.singleton.Cache;

/**
 * @author - Saori Sugiyama
 * @contact - sugiyama.saori.biz@gmail.com
 * @date - 4/20/17
 */

public class SplashActivity extends com.astir_trotter.atcustom.ui.activity.SplashActivity {

    @Nullable
    @Override
    protected Class<?> getNextActivityClass() {
        return MainActivity.class;
    }

//    @Override
//    protected long getDelayDuration() {
//        return 1000;//super.getDelayDuration(); // TESTCODE
//    }
}

File: projects/DemoApp/app/src/main/java/com/astir_trotter/atcustom/demoapp/activity/SplashActivity.kt
package com.astir_trotter.atcustom.demoapp.activity

import com.astir_trotter.atcustom.demoapp.activity.main.MainActivity

/**
 * @author - Saori Sugiyama
 * *
 * @contact - sugiyama.saori.biz@gmail.com
 * *
 * @date - 4/20/17
 */

class SplashActivity : com.astir_trotter.atcustom.ui.activity.SplashActivity() {

    override fun getNextActivityClass(): Class<*>? {
        return MainActivity::class.java
    }

    override fun getDelayDuration(): Long {
        return 1000 // TESTCODE
        //        return super.getDelayDuration();
    }
}
